presto-parser/src/main/java/com/facebook/presto/sql/SqlFormatter.java

```diff
@@ -924,9 +939,7 @@ protected Void visitAddColumn(AddColumn node, Integer indent)
             builder.append("ALTER TABLE ")
                     .append(node.getName())
                     .append(" ADD COLUMN ")
-                    .append(node.getColumn().getName())
-                    .append(" ")
```

Comment:
It looks like you fixed a bug here, can you add a test for that (possibly extract as separate commit). 

Bug is about adding a column with `COMMENT`, old code was losing `COMMENT` here.